integration_test.go

```diff
@@ -916,3 +916,81 @@ func TestConvertStdin(t *testing.T) {
 
 	}
 }
+
+type validateTest struct {
+	arguments      []string
+	source         string
+	expectError    bool
+	expectedResult string
+}
+
+var validateTests = []validateTest{
+	{
+		arguments:      []string{"validate", "-a", "6.2.0", filepath.Join(sourcePath, "pmem_cluster_cr.yaml")},
+		expectError:    false,
+		expectedResult: "",
+		source:         filepath.Join(sourcePath, "pmem_cluster_cr.yaml"),
+	},
+	{
+		arguments:   []string{"validate", "-a", "7.0.0", filepath.Join(extraTestPath, "server64/server64.yaml")},
+		expectError: true,
+		source:      filepath.Join(extraTestPath, "server64/server64.yaml"),
+		expectedResult: `context: (root).namespaces.0
+	- description: Additional property memory-size is not allowed, error-type: additional_property_not_allowed
+context: (root).namespaces.0.storage-engine
```

Comment:
If only we could tell them what namespace it is. Not just the index. I know this is likely in the json schema validation code but it might be pretty easy to replace `0` with `test`. Some users have a large number of namespaces and sets.

Reply:
Sets and DC namespaces might be a little more complex though

Reply:
I'd like to do this but the work required is complex enough that I chose to push it out.

Reply:
https://aerospike.atlassian.net/browse/TOOLS-2765